vendor/github.com/ligato/cn-infra/logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -119,7 +119,7 @@ func (p *Plugin) Init() error {
 //   > curl -X PUT http://localhost:<port>/log/<logger-name>/<log-level>
 func (p *Plugin) AfterInit() error {
 	if p.HTTP != nil {
-		p.HTTP.RegisterHTTPHandler(fmt.Sprintf("/log/{%s}/{%s:debug|info|warn|error|fatal|panic}",
+		p.HTTP.RegisterHTTPHandler(fmt.Sprintf("/log/{%s}/{%s}",
```

Comment:
have you upstreamed these changes to ligato/cn-infra ? Otherwise they will be overwritten by next dependency update.

Reply:
No. These changes appear in the newer version of cn-infra. When i prepared commits for this PR, i took the newest version of cn-infra as target for resolving vendor differences, but i should have taken the version used in Contiv's dev branch to stay in sync with output of dep tool. Fixing it in https://github.com/contiv/vpp/pull/1663/commits/6f4bd149aa3146ab7cd42a2d36a55f6147253898.

Reply:
PS: the change is [fix](https://github.com/ligato/cn-infra/commit/b24f34014e533a82a1c744e84b1a5367ff5ad9b6#diff-5ccefe7767d91cafe6fa6ecf8c385132) in cn-infra for unrelated stuff. It fixes possible logging problems for vpp-agent tools where you can set logging by setting string ("debug", "warn",...) , so it should not change working/notworking status of functionality in SRv6 (and i think in Contiv in general).